src/components/Popover/Popover.tsx

```diff
@@ -73,11 +73,9 @@ export const defaultPopoverModifiers = [
 /**
  * BETA: This component is still a work in progress and is subject to change.
  *
- * A styled popover built on 'headless UI' popover. Consider using the Dropdown component, instead.
```

Comment:
I moved this line down because the import should be at the top of the comment (unless there's a BETA line, in which case that one's at the top).